hs_core/templates/search/search.html

```diff
@@ -2,7 +2,8 @@
 {% load geoanalytics_tags pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
 {% block title %}Discovery{% endblock %}
 {% block main %}
-<div class="container" id="main">
+<div id="discover-load" style="display: none"></div>
```

Comment:
I think it's better to use the icons we have in the Font Awesome library in order to improve re-usability. We could use this instead which doesn't require an image file and is the same one I'm using in the file browser code I'm working on:
`<i class="fa fa-spinner fa-spin fa-3x fa-fw"></i> `

see http://fontawesome.io/examples/#animated